### PR TITLE
[IOTDB-3432] Add IT for raw query

### PR DESCRIPTION
**Pull request by JackieTien97**

### CODE REVIEW

**Review by HTHou — APPROVED** [review]

**Comment by ericpai** [comment]
@JackieTien97 Please remember removing old IT files once the github action for new IT is ready

File: integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "insert into root.sg1.d2(time, s2, s5) values(40, 40, 'non_aligned_test40')",
+      };
+
+  public static void insertData() throws ClassNotFoundException {
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+
+      // create aligned and non-aligned time series
+      for (String sql : sqls) {
+        statement.execute(sql);
```
Comment (ericpai) [inline integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java:140]: Please use `executerBatch` instead to accelerate the insert procedure.

**Review by ericpai — CHANGES_REQUESTED** [review]

File: integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java (hunk 1 above)
Comment (JackieTien97) [reply integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java:140]: `executeBatch` has not been implemented in new cluster yet.

Comment (ericpai) [reply integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java:140]: So just add TODO comments for future

Comment (JackieTien97) [reply integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java:140]: ok

**Review by ericpai — APPROVED** [review]

File: integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail();
```
[the remark this answers is not in the document]
Comment (JackieTien97) [reply integration-test/src/test/java/org/apache/iotdb/db/it/aligned/AlignedWriteUtil.java]: fixed

**Pull request merged**